charts/opentelemetry-demo-datadogv2/exporter.yaml

```diff
@@ -0,0 +1,66 @@
+ingest: false
+opentelemetry-demo:
+  opentelemetry-collector:
+    mode: "daemonset"
+    config:
+      receivers:
+        otlp:
+          protocols:
+            http:
+            grpc:
+        # The hostmetrics receiver is required to get correct infrastructure metrics in Datadog.
+        hostmetrics:
+          collection_interval: 10s
+          scrapers:
+            paging:
+              metrics:
+                system.paging.utilization:
+                  enabled: true
+            cpu:
+              metrics:
+                system.cpu.utilization:
+                  enabled: true
+            disk:
+            filesystem:
+              metrics:
+                system.filesystem.utilization:
+                  enabled: true
+            load:
+            memory:
+            network:
+            processes:
+        # The prometheus receiver scrapes metrics needed for the OpenTelemetry Collector Dashboard.
+        prometheus:
+          config:
+            scrape_configs:
+              - job_name: 'otelcol'
+                scrape_interval: 10s
+                static_configs:
+                  - targets: [ '0.0.0.0:8888' ]
+
+      processors:
+        batch:
+          # Datadog APM Intake limit is 3.2MB. Let's make sure the batches do not
+          # go over that.
+          send_batch_max_size: 1000
+          send_batch_size: 100
+          timeout: 10s
+
+      exporters:
+        datadog:
+          api:
+            site: datadoghq.com
+            key: ${DD_API_KEY}
+          host_metadata:
+            tags:
+              - "env:otel-exporter"
+      service:
```

Comment:
Can we logs pipelines ? see https://github.com/DataDog/opentelemetry-helm-charts/blob/e7404a0c87dae9f5b6fc8e2cd84f29ed9f25d19c/charts/opentelemetry-demo-datadog/otelcol-config.yaml#L185